modules/swagger-parser-v3/src/test/resources/issue-1761.yaml

```diff
@@ -0,0 +1,214 @@
+openapi: 3.0.3
+info:
+  title: openapi 3.0.3 sample spec
+  description: sample spec for testing openapi functionality, built from json schema
+    tests for draft6
+  version: 0.0.1
+paths: {}
+components:
+  schemas:
+    SimpleEnumValidation:
+      enum:
+        - 1
+        - 2
+        - 3
+    HeterogeneousEnumValidation:
+      enum:
+        - 6
+        - foo
+        - []
+        - true
+        - foo: 12
+    HeterogeneousEnumWithNullValidation:
+      enum:
+        - 6
+        - null
+    EnumsInProperties:
+      type: object
+      properties:
+        foo:
+          enum:
+            - foo
+        bar:
+          enum:
+            - bar
+      required:
+        - bar
+    EnumWithEscapedCharacters:
+      enum:
+        - 'foo
+
+        bar'
+        - "foo\rbar"
+    EnumWithFalseDoesNotMatch0:
+      enum:
+        - false
+    EnumWithTrueDoesNotMatch1:
+      enum:
+        - true
+    EnumWith0DoesNotMatchFalse:
+      enum:
+        - 0
+    EnumWith1DoesNotMatchTrue:
+      enum:
+        - 1
+    NulCharactersInStrings:
+      enum:
+        - "hello\0there"
+  x-schema-test-examples:
```

Comment:
You probably don't need this vendor extension test data because your project is not using it